<ui version="4.0" >
 <class>DlgContentFetchSettingWidget</class>
 <widget class="QWidget" name="DlgContentFetchSettingWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>521</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="scriptGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>User Scripts</string>
     </property>
     <layout class="QGridLayout" >
      <item row="1" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="newScriptButton" >
          <property name="text" >
           <string>New Script...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configureScriptButton" >
          <property name="text" >
           <string>Configure...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editScriptButton" >
          <property name="text" >
           <string>Edit...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeScriptButton" >
          <property name="text" >
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>91</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QTreeWidget" name="scriptTreeWidget" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Path</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>RegExp</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Description</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
